product-service/.serverless/cloudformation-template-update-stack.json

```diff
@@ -167,13 +179,22 @@
           "S3Bucket": {
             "Ref": "ServerlessDeploymentBucket"
           },
-          "S3Key": "serverless/product-service/dev/1616594074692-2021-03-24T13:54:34.692Z/product-service.zip"
+          "S3Key": "serverless/product-service/dev/1618773377485-2021-04-18T19:16:17.485Z/product-service.zip"
         },
         "Handler": "handler.getProducts",
         "Runtime": "nodejs12.x",
         "FunctionName": "product-service-dev-getProducts",
         "MemorySize": 1024,
         "Timeout": 6,
+        "Environment": {
+          "Variables": {
+            "PG_HOST": "products-instance.ck2amayvjttv.eu-west-1.rds.amazonaws.com",
```

Comment:
Please do not commit your credentials -it's very unsecure, you should understand it
At least use dotenv package